tst/CTA.WebForms2Blazor.Tests/FileInformationFactoryTests.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using System.IO;
+using NUnit.Framework;
+
+namespace CTA.WebForms2Blazor.Tests
+{
+    [TestFixture]
+    class FileInformationFactoryTests
+    {
+        private const string TEST_FILES_DIRECTORY_PATH = "TestingArea/TestFiles";
+
+        private string _testProjectPath;
+        private string testCodeFilePath;
+        private string testConfigFilePath;
+        private string testStaticFilePath;
+        private string testViewFilePath;
+        private string testProjectFilePath;
```

Comment:
nit 1: Rename constant to use pascal case (TestFilesDirectoryPath) to remain consistent with our team's adopted style

nit 2: rename private class variables to "_variableName" for consistency